base/src/com/google/idea/blaze/base/run/BlazeBeforeRunCommandHelper.java

```diff
@@ -65,7 +67,7 @@ private BlazeBeforeRunCommandHelper() {}
         (BlazeCommandRunConfigurationCommonState) configuration.getHandler().getState();
     WorkspaceRoot workspaceRoot = WorkspaceRoot.fromProject(project);
     ProjectViewSet projectViewSet = ProjectViewManager.getInstance(project).getProjectViewSet();
-
+      ServiceManager.getService(ImportProblemContainerService.class).resetIssues();
```

Comment:
@hmemcpy - is there an event system in Intellij such that instead of triggering an action here, we can broadcast an event that will cause other processes to be triggered?